java-frontend/src/main/java/org/sonar/java/resolve/JavaSymbol.java

```diff
@@ -400,6 +400,18 @@ public String getFullyQualifiedName() {
       return builder.build();
     }
 
+    @CheckForNull
+    JavaType firstEnclosingClass() {
```

Comment:
Not required as you can rely on owner.enclosingClass to achieve the same purpose